src/components/Editor/QuillEditor.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useEffect, useRef, useState } from "react";
+import "../../css/quillEditor.css";
+import { useFirebase } from "react-redux-firebase";
+import { useDispatch, useSelector } from "react-redux";
+import { Prompt } from "react-router-dom";
+import { setCurrentStep } from "../../store/actions";
+import * as Y from "yjs";
+import { QuillBinding } from "y-quill";
+import Quill from "quill";
+import QuillCursors from "quill-cursors";
+import { FirestoreProvider, getColor } from "@gmcfall/yjs-firestore-provider";
+
+import { onlineFirebaseApp } from "../../config";
+
+Quill.register("modules/cursors", QuillCursors);
+
+const QuillEditor = ({ id, data, tutorial_id }) => {
+  const [allSaved, setAllSaved] = useState(true);
+  const editorRef = useRef(null);
+  const containerRef = useRef(null);
+  let noteID = id || "test_note";
+  const firebase = useFirebase();
+  const dispatch = useDispatch();
+  const basePath = ["cl_codelabz", "organization", "codelabzorg", tutorial_id];
+  var provider, binding;
+  var ydoc;
+
+  const currentUserHandle = useSelector(
+    ({
+      firebase: {
+        profile: { handle }
+      }
+    }) => handle
+  );
+
+  useEffect(() => {
+    setAllSaved(true);
+    // const confirmBeforeExit = (e) => {
+    //   e.preventDefault();
+    //   e.returnValue = "";
+    // };
+    // window.addEventListener("beforeunload", confirmBeforeExit);
+
+    return () => {
+      // window.removeEventListener("beforeunload", confirmBeforeExit);
```

Comment:
Hi, can you remove the commented-out code? We can do if it does not serve any purpose.